egs/tedlium/s5_r2_wsj/local/tuning/run_segmentation_wsj_d.sh

```diff
@@ -0,0 +1,146 @@
+#! /bin/bash
+
+. ./cmd.sh
+. ./path.sh
+
+set -e -o pipefail -u
+
+segment_stage=-13
+affix=_1d
+decode_nj=30
+cleanup_stage=-1
+
+###############################################################################
+# Segment long recordings using TF-IDF retrieval of reference text 
+# for uniformly segmented audio chunks based on Smith-Waterman alignment.
+# Use a model trained on train_si84 (tri2b)
+###############################################################################
+
+###
+# STAGE 0
```

Comment:
There are results after each stage. I think the script is very complicated and is easier to leave it like most run scripts that don't have stages.